src/main/web_src/src/components/bestilling/sammendrag/kriterier/BestillingKriterieMapper.js

```diff
@@ -915,7 +915,7 @@ export function mapBestillingData(bestillingData, bestillingsinformasjon) {
 				items: [
 					obj(
 						'Avgjørelsesdato',
-						Formatters.formatDate(
+						Formatters.formatStringDates(
```

Comment:
Ah, ser ut til at vi har ulikt datoformat på bestillingsoppsummeringen på steg 3 og i bestillingslisten! For nå feiler alt på steg 3 😅  Jeg hadde akkurat samme problem på org., og da endte jeg opp med å bytte tilbake til `Formatters.formatDate`, og i `Formatters.formatDate` fjernet jeg denne sjekken: `if (date.length == 10) return date`. Vet ikke hva den er godt for, så forhåpentligvis ødelegger det ikke for noe annet 😉

Reply:
Åja skjønner 😅  Hehe, tenkte på å gjøre det, men så jo ut som StringDates skulle gjøre jobben 😂 
Ja supert, da gjør jeg det 😄